src/software/world/ball_state.cpp

```diff
@@ -29,5 +29,5 @@ bool BallState::operator==(const BallState &other) const
 
 bool BallState::operator!=(const BallState &other) const
 {
-    return false;
+    return !(*this == other);
```

Comment:
:eyes: :heavy_check_mark:

Reply:
ya :eyes:... my bad from the BallState issue

Reply:
All good! We've all put in fixes like this :laughing: